Pa3/Durchführung/Project/src/app/State/DriveOverGap.c

```diff
@@ -15,7 +15,8 @@
 #include "service/LineSensor.h"
 
 /* CONSTANTS **************************************************************************************/
-
+#define STATE_IS_ACTIVE_FOR_TOO_LONG 5000
```

Comment:
This and next line: codingstyle wants brackets
Also: A name like STATE_TIME_MAX_MS makes more sense to me